src/smtpclient.cpp

```diff
@@ -369,34 +377,109 @@ bool SmtpClient::sendMail(MimeMessage& email)
             sendMessage("RCPT TO:<" + (*it)->getAddress() + ">");
             waitForResponse();
 
-            if (responseCode != 250) return false;
```

Comment:
Here, I would let the current `sendMail()` call to fail and let the caller implement his own fallback strategy. In this way the caller could decide to what he wants to do: try to correct the recipient list *(using VRFY)*, send the email to each recipient separately *(let if fail for the incorrect ones)* or just show an error message.

Reply:
Well, this approach has a disadvantage that the actual composition of the mail message must be started again ... If you have many RCPTs (from database records, for example), you will need RSET and start again. Returning the records from the database.
A possible solution can be a member-variable for configuring this behavior (return an, error, correct email or show a message)...

Reply:
Well, this approach has a disadvantage that the actual composition of the mail message must be started again ... If you have many RCPTs (from database records, for example), you will need RSET and start again. Returning the records from the database.
A possible solution can be a member-variable for configuring this behavior (return an error, correct email or show a message)...